### ISSUE TITLE
Is diag operator is same as "DistMult" Model ?

### ISSUE THREAD

**Issue by jaiabhayk** [facebookresearch/PyTorch-BigGraph#219]
is operator "diagonal" explained in (https://torchbiggraph.readthedocs.io/en/latest/scoring.html#operators) is same as DistMult (https://arxiv.org/abs/1412.6575) .

Because documentation mentions nowhere about this but claims to support (which is confusing):

`The generality and extensibility of its model allows PBG to train a number of models from the knowledge graph embedding literature, including TransE, RESCAL, DistMult and ComplEx.`

**Comment by lw** [comment]
A model design (such as DistMult) is more than just the operator: it also includes the loss function, the comparator, whether bias is used, ... I don't remember the specifics of DistMult, so you'll need to see what building blocks you'll need to assemble to obtain it.

**Comment by lw** [comment]
BTW Have you checked out this? https://torchbiggraph.readthedocs.io/en/latest/related.html#distmult

**Comment by jaiabhayk** [comment]
Thanks @lw  .
